src/parameterized/gamma.jl

```diff
@@ -0,0 +1,27 @@
+# Gamma distribution
+
+export Gamma
+
+@parameterized Gamma(α,β) ≪ Lebesgue(ℝ₊)
+
+@kwstruct Gamma(α, β)
+
+@kwalias Gamma [
+    a     => α
+    alpha => α
+    b     => β
+    beta  => β
+]
+
+TV.as(::Gamma) = asℝ₊
+
+function logdensity(d::Gamma{(:α, :β)}, x)
+    return d.α * log(d.β) + (d.α - 1) * log(x) - d.β * x - loggamma(d.α)
+end
+
+Base.rand(rng::AbstractRNG, T::Type, μ::Beta) = rand(rng, Dists.Gamma(μ.α, 1/μ.β))
```

Comment:
I think this isn't needed if we already have `distproxy` (but let's check that this is the case)

Reply:
Confirmed, let's drop this line